lib/zoom/actions/webinar.rb

```diff
@@ -28,6 +28,9 @@ module Webinar
                             allow_anonymous_questions answer_questions attendees_can_comment
                             attendees_can_upvote enable
                           ]
```

Comment:
```suggestion
                          ],
```
I missed a comma. I'll clean it up though.